binary_transparency/firmware/README.md

```diff
@@ -119,16 +119,20 @@ Record the tree ID that is returned by the command above, it will be referred to
 as $TREE_ID by subsequent commands:
 
 #### Terminal 2 - FT Personality:
-* Open terminal in root of `firmware-transparency-demo` git repo, run:
+* Open a terminal in the `binary_transparency/firmware` directory.
+* A file is needed to hold the CAS DB which will back the log, this file
+  needs to be available for the duration of this log, so writing to '/tmp'
+  is considered risky. Choose a file path and add as below.
 
 ```bash
-go run ./cmd/ft_personality/main.go --logtostderr -v=2 --tree_id=$TREE_ID --cas_db_file=/path/to/ft.db
+export CAS_DB_FILE='/full/path/to/file.db'
+go run ./cmd/ft_personality/main.go --logtostderr -v=2 --tree_id=$TREE_ID --cas_db_file=${CAS_DB_FILE}
 ```
 
 The `cas_db_file` needs to be available for the duration of this log, so writing to `/tmp/` is considered risky.
```

Comment:
Now dup'ed into comment above

Reply:
d'oh, my manual `patch -p1` is clearly faulty :)
Thanks, fixed.